config/makeprops.py

```diff
@@ -64,7 +66,7 @@ class Language(StrEnum):
         const int length;
     };
 
-    class PropertyNames
+    class ICE_API PropertyNames
```

Comment:
We now export these to access in IceStorm. Once we fix https://github.com/zeroc-ice/ice/issues/2798 this can go away.